assets/src/api.ts

```diff
@@ -26,6 +26,13 @@ export type RegisterParams = LoginParams & {
   passwordConfirmation: string;
 };
 
+export type WidgetConfigParams = {
+  title: string;
+  subtitle: string;
+  color: string;
+  account_id: string;
```

Comment:
we don't need to pass this in explicitly since we can just get it from `conn.assigns.current_user.account_id` in the controller right?